### PR TITLE
add icons and coloured quotes to get help

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/WdEHHBTi/442-eee-as-a-woman-who-felt-calmed-by-the-front-page-i-need-the-rest-of-the-pages-to-feel-just-as-friendly-so-that-i-dont-lose-trust

## Description

- add icons and coloured quotes to get help
- this doesn't include extra images on the grid page

image

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -755,6 +755,9 @@ t key =
         GetHelpSection1Title ->
             "Support from survivors"
 
+        GetHelpSection1Icon ->
```
Comment (katjam) [inline src/Copy/Text.elm:758]: I thought we decided we didn't think these icons worked with the rest if the design?

File: src/View/GetHelp.elm [hunk 2]
```diff
@@ -83,13 +84,17 @@ columnStyle =
 -- This function is a little bit eek!
 
 
-card : Float -> String -> String -> String -> CallToAction -> Html msg
-card viewportWidth title quote description call =
+card : Float -> String -> String -> String -> String -> String -> CallToAction -> Html msg
```
Comment (katjam) [inline src/View/GetHelp.elm:87]: Super unhappy with this number of args, but :face_with_head_bandage:

**Review by katjam — DISMISSED** [review]
Sure if we want to keep the icons, this looks good to go it. What's the test fail?

**Comment by katjam** [comment]
Afterthought: Can you stick Support&nbsp;Line topgether?

**Comment by katjam** [comment]
Screenshot_20201007_183500

**Comment by Nikomus** [comment]
image

**Review by katjam — APPROVED** [review]
Nice! :police_car:

**Pull request merged**